src/redux/store.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/booksSlice';
+import categoryReducer from './categories/categoriesSlice';
+
+const store = configureStore({
+  reducer: {
+    book: bookReducer,
+    category: categoryReducer,
+  },
+});
+
+export default store;
```

Comment:
Good job on the store  🏬 but the main React component should be wrapped with the `<Provider>` component from `react-redux` and the Redux `store` has been passed into the `<Provider>`, a little bit like that:
```
import React from "react"
import ReactDOM from "react-dom/client"
import { Provider } from "react-redux"
import store from "./redux/store"

ReactDOM.render(
  <Provider store={store}>
    <App />
  </Provider>,
  document.getElementById("root")
)
```